src/Chainweb/Version.hs

```diff
@@ -209,109 +206,76 @@ instance IsMerkleLogEntry ChainwebHashTag ChainwebVersion where
     {-# INLINE fromMerkleNode #-}
 
 chainwebVersionToText :: ChainwebVersion -> T.Text
-
--- production versions
-chainwebVersionToText Testnet00 = "testnet00"
-
--- test versions
-chainwebVersionToText v@Test{} = "test-" <> sshow (chainwebVersionId v)
-chainwebVersionToText v@TestWithTime{} = "testWithTime-" <> sshow (chainwebVersionId v)
-chainwebVersionToText v@TestWithPow{} = "testWithPow-" <> sshow (chainwebVersionId v)
-chainwebVersionToText v@Simulation{} = "simulation-" <> sshow (chainwebVersionId v)
+chainwebVersionToText v =
+    case HM.lookup v prettyVersions of
+        Just t -> t
+        Nothing -> case v of
+            Test{} -> "test"
+            TestWithTime{} -> "testWithTime"
+            TestWithPow{} -> "testWithPow"
+            Testnet00 -> "testnet00"
```

Comment:
Same as below: I don't see a reason to move the production versions and "default" versions into this case. In particular the production cases should be separate cases, which is more efficient and more readable.

Reply:
I believe the production versions are already caught somewhere higher up the call chain. Adding the case here was only to prevent exhaustive pattern match warnings.